[SRW-AQM] Build Errors on Hera after Rocky8 Update

## Expected behavior
UFS-SRW-App should compile on Hera with [aqm_dev] branch.

## Current behavior
UFS-SRW-App does not compile on Hera with [aqm_dev] branch after the recent Rocky8 update.

[ 52%] Building Fortran object FV3/ccpp/physics/CMakeFiles/ccpp_physics.dir/physics/samfshalcnv.f.o
[ 53%] Linking Fortran static library libupp.a
/usr/bin/ar: Relink `/apps/oneapi/compiler/2022.0.2/linux/compiler/lib/intel64_lin/libimf.so' with `/lib64/libm.so.6' for IFUNC symbol `sinf'
Error running link command: Segmentation faultmake[5]: *** [FV3/upp/sorc/ncep_post.fd/CMakeFiles/upp.dir/build.make:2167: FV3/upp/sorc/ncep_post.fd/libupp.a] Error 1
make[5]: *** Deleting file 'FV3/upp/sorc/ncep_post.fd/libupp.a'
...
...
...
[ 60%] Linking Fortran static library libccpp_physics.a
/usr/bin/ar: Relink `/apps/oneapi/compiler/2022.0.2/linux/compiler/lib/intel64_lin/libimf.so' with `/lib64/libm.so.6' for IFUNC symbol `sinf'
Error running link command: Segmentation faultmake[5]: *** [FV3/ccpp/physics/CMakeFiles/ccpp_physics.dir/build.make:2722: FV3/ccpp/physics/libccpp_physics.a] Error 1
make[5]: *** Deleting file 'FV3/ccpp/physics/libccpp_physics.a'

## Machines affected
Hera

## Steps To Reproduce
Note:  This is still following AQM Wiki for [aqm_dev] branch:  https://github.com/ufs-community/ufs-srweather-app/wiki/Air-Quality-Modeling

git clone -b aqm_dev https://github.com/ufs-community/ufs-srweather-app.git
   cd ufs-srweather-app/sorc

Update the ../modulefiles/build_hera_intel.lua for latest CMake to compile.  See PR #1056 
 
./app_build.sh -p=hera -a=ATMAQ --extrn

--- a/modulefiles/build_hera_intel.lua
+++ b/modulefiles/build_hera_intel.lua
@@ -5,7 +5,7 @@ the NOAA RDHPC machine Hera using Intel-2022.1.2
 
 whatis([===[Loads libraries needed for building the SRW workflow on Hera ]===])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env-noavx512/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env-rocky8/install/modulefiles/Core")
 prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
 
 load(pathJoin("stack-intel", "2021.5.0"))
@@ -39,7 +39,6 @@ load(pathJoin("g2tmpl", "1.10.2"))
 load(pathJoin("ip", "4.3.0"))
 load(pathJoin("sp", "2.3.3"))
 load(pathJoin("gftl-shared", "1.5.0"))
-load(pathJoin("yafyaml", "0.5.1"))
 load(pathJoin("bufr", "12.0.0"))
 load(pathJoin("gfsio", "1.4.1"))
 load(pathJoin("landsfcutil", "2.4.1"))

